sdk/appconfiguration/app-configuration/sample-react/tsconfig.json

```diff
@@ -1,11 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
+    "lib": ["dom", "dom.iterable", "esnext"],
```

Comment:
The sample doesn't need a target?

Reply:
whoops this is a bug, I excluded 'samples' but this is `sample-react` one sec will fix